import_map.json

```diff
@@ -5,7 +5,7 @@
   "scopes": {
     "examples/": {
       "capi": "./mod.ts",
-      "capi/": "./",
+      "capi/patterns/": "./patterns/",
```

Comment:
Why this change?

Reply:
Otherwise we get a warning on importing from `"capi"` (wants to redirect to `"capi/patterns/..."`)